packit.spec

```diff
@@ -71,6 +71,7 @@ python3 setup.py --command-packages=click_man.commands man_pages --target %{buil
 %if 0%{?fedora} >= 30
 %{_mandir}/man1/packit*.1*
 %endif
+%{_datadir}/bash_completion/completions/%{real_name}
```

Comment:
* You have a typo there `bash_completion` -> `bash-completion`
* Since you don't have `bash-completion` in `Requires:`  (which is OK, since `packit` doesn't require it to work) you should not rely on the directory to be there - and create it yourself if it's not there:


```spec
%dir %{_datadir}/bash-completion/completions
%{_datadir}/bash-completion/completions/%{real_name}
```